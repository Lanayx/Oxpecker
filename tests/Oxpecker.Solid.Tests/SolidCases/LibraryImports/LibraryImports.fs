module Oxpecker.Solid.Tests.Cases.LibraryImports

open Oxpecker.Solid
open Oxpecker.Solid.Tests.Cases.Types


[<SolidComponent>]
let Button () =
    ImportedTag(class'="TEST").attr("key","value")
