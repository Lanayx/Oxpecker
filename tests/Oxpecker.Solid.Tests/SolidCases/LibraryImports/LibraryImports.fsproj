<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Types.fs" />
    <Compile Include="LibraryImports.fs" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\..\..\..\src\Oxpecker.Solid\Oxpecker.Solid.fsproj" />
  </ItemGroup>

</Project>
