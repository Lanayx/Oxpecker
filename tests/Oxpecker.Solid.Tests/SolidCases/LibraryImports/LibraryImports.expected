import { Faketag } from "fakemodule";

export function Button() {
    return <Faketag class="TEST"
        key="value"></Faketag>;
}

